import/src/test/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreServiceTest.java

```diff
@@ -76,7 +76,7 @@ private int createDicomServer(boolean connectError, int responseCode) throws Exc
     IDicomWebClient dicomWebClient =
```

Comment:
Can we add a couple unit tests for redaction?

Reply:
Redactor is unit tested in it's own project, so I don't think it would be a good idea to duplicate it in adapter.
I could add it to integration test though.

Reply:
Makes sense, let's add it to the integration test.

Reply:
Will finish it on monday, probably.

Reply:
Updated integration test to use redactor, this involves few extra changes to work around UID replacement that redactor does by default (not configurable). Also organized integration test files into scripts and data subfolders.